### PR TITLE
Add eqV2_tracegrad Model

### PR DESCRIPTION
**Pull request by TheSheng1103**
Hello Matbench Discovery Team,

First and foremost, I would like to express my sincere gratitude to you for your incredible efforts in building and maintaining the Matbench Discovery benchmark.Your work has enabled the machine learning models in the field of materials science to continuously progress and improve.

Our team recently trained a conservative model (eqV2_tracegrad) on the MPtrj training set, and we believe it may enhance the performance of the tasks you listed. We would be delighted to incorporate it into your benchmark tests.

I will add all the required files outlined in the contribution guide, please let me know if you need further information.

The submission also includes:

- a CLI to merge predictions, compute formation energies with optional MP-2020 corrections, and export CSV/JSON and large-error reports
- pipelines for geometry-optimized discovery outputs and WBM relaxation energies
- a parallel kappa workflow with reproducible runs, per-structure diagnostics, and merged results
- shell scripts for discovery, kappa, and aggregation workflows

Please open PRs as `draft` and only mark as `ready to review` after checking off the following items:

- [ ] I created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [ ] I added the new model as a new attribute [`Model.<model_variant>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) on the `Model` enum in `enums.py`.
- [ ] I uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [ ] I uploaded the model-relaxed structures file to Figshare or another cloud storage service in [JSON lines format](https://jsonlines.org) (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.jsonl.gz`). JSON Lines allows fast loading of small numbers of structures with `pandas.read_json(lines=True, nrows=100)` for inspection.
- [ ] I uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [ ] I have uploaded the diatomic predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-diatomics.json.gz`).
- [ ] I included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [ ] I included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.
- [ ] I have run `just prepare-model-submission <model_variant>` as described in the [contributing guide](https://github.com/janosh/matbench-discovery/blob/main/contributing.md) to check metadata and generate the plots needed for submission.
- [ ] I have installed/run the pre-commit hooks (`prek install` or `uvx prek`) and ensured all checks are passing.

Marking your PR as `ready to review` will trigger an automated code review, please address any issues raised by the automated review. For the maintainers minimizing the final human review process enables us to merge your submissions much faster!

- [ ] I included a training script (`train_<arch_name>.py`) if I trained a model specifically for this benchmark.
- [ ] I included a `readme.md` with additional details about my model.

### CODE REVIEW

**Comment by janosh** [comment]
thanks for the submission but closing this as stale for now. feel free to ping back!
